### ISSUE TITLE
I can not reproduce your work

### ISSUE THREAD

**Issue by ANDYLLY** [ChickenEating/IR-Enhancement#3]
Can you provide the corresponding .pth file? Futhermore, your train.py can not work normally, althrough l have try my best to fix the potential problems. 

For example,  in DAU.py: center_pixel_value = self.conv(x)[:, :, 1, 1].unsqueeze(2).unsqueeze(3), 
it seems that self.conv is a convolution with size (1, 1, 3, 3) but the data is (bn, 64, H, W). 

Moreover, in CIM.py: concatenated_detail = torch.cat(high_freq, detail_feature, dim=1), 
l think the grammar for torch.cat is not correct, the first argument should be put into a list. 

And in detail_enhanced = self.detail_enhancer(concatenated_detail), 
to process the data cat in channel, which means that mean should at least be 2, the first convolution in self.detail_enhancer is self.detail_enhancer = nn.Sequential(
            nn.Conv2d(1, 64, kernel_size=3, padding=1),
            nn.ReLU(inplace=True),
            nn.Conv2d(64, 1, kernel_size=3, padding=1),
            nn.ReLU(inplace=True)
        ), this can not work at all ????

**Comment by ANDYLLY** [comment]
Besides, the figure on experiment result in your paper for FLIR dataset is confusing. I have used FLIR dataset for sometime, it seems the low quality or the image need to be enhanced is the image released in FLIR dataset. But in your paper, you consider the images in FLIR dataset as high quality images, and use random contrast and adding noise to generate low quality images. So the images seem to be wired. In other words, the last subfigure in Figure 6 is not the original image in FLIR dataset. Is this correct?
